ClientSideEncryptionToCPKN/ClientSideKeyVaultKeyToCustomerManagedKey/README.md

```diff
@@ -0,0 +1,29 @@
+# Key Vault Client Side Encryption to Customer Managed Key Server Side Encryption
+Migration Sample From Key Vault Client Side Encryption to Customer Managed Key Server Side Encryption
+
+## General Info
+This sample has an optional setup process that uploads an example client side encrypted blob (blobExample.txt in samplesetup folder) and uploads it to a newly created container in the provided storage account.
+The sample then migrates the blob to server side encryption using a customer managed key from Azure Key Vault and uploads the server side encrypted blob in the same container. 
+The key used for server side encryption will be the same key used for client side encryption.
+
+## Prerequisites
+Requires installation of .NET and [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest).
```

Comment:
Might want to add link to .NET downloads. Also please mention we're looking for `NET Core`.